gamemodes/terrortown/entities/entities/ttt_flame.lua

```diff
@@ -0,0 +1,240 @@
+-- fire handler that does owned damage
+
+AddCSLuaFile()
+
+ENT.Type = "anim"
+ENT.Model = Model("models/weapons/w_eq_flashbang_thrown.mdl")
+
+---
+-- @accessor Player
+-- @realm shared
+AccessorFunc(ENT, "dmgparent", "DamageParent")
+
+---
+-- @accessor boolean
+-- @realm shared
+AccessorFunc(ENT, "die_explode", "ExplodeOnDeath")
+
+---
+-- @accessor number
+-- @realm shared
+AccessorFunc(ENT, "dietime", "DieTime")
+
+ENT.firechild = nil
+ENT.fireparams = {
+	size = 120,
+	growth = 1
+}
+
+ENT.next_hurt = 0
+ENT.hurt_interval = 1
+
+---
+-- @realm shared
+function ENT:SetupDataTables()
+	self:NetworkVar("Bool", 0, "burning")
```

Comment:
```suggestion
	self:NetworkVar("Bool", 0, "Burning")
```
otherwise the Accessor will be `Setburning()` instead of `SetBurning()`